fabfile.py

```diff
@@ -4,6 +4,20 @@
 from db import cache
 import config
 
+import os
+
+def vpsql():
+    """ Connect to the acousticbrainz database running on vagrant """
+    local("psql -h localhost -p 15432 -U postgres acousticbrainz")
+
+def vssh():
+    """ SSH to a running vagrant host """
```

Comment:
I forgot to nitpick to your docstrings. https://www.python.org/dev/peps/pep-0257/#one-line-docstrings :octopus: